README.md

```diff
@@ -23,6 +23,9 @@ You can find the latest stable version here:
 
 [![GitHub All Releases](https://img.shields.io/github/downloads/Open-Shell/Open-Shell-Menu/total?style=for-the-badge&color=4bc2ee&logo=github)](https://github.com/Open-Shell/Open-Shell-Menu/releases/latest)
 
+### ⚠️ WARNING FOR WINDOWS ARM USERS ⚠️
```

Comment:
What about just `Warning for Windows for ARM users`?
Without the rather annoying all caps?
I guess those exclamation icons are enough to draw the attention.

Reply:
Or maybe just `Windows for ARM compatibility` ?

Reply:
Sure, I can change the title to be  less intense! It bricked my VM so I was a bit agitated at the time 😛

Reply:
We should add a few checks for the installer to only work on tested environments. There's been a few reports of the installer doing bad things to recent versions of windows.